.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+
+before_script:
+  - npm install -g aglio
+
+script:
+  - cd docs
+  - aglio -i ./auth.apib -o auth.html
```

Comment:
Instead of having multiple pages for the API, I'd suggest having an "index" file that import the others - that way the rendered version will contain all related APIs on a single page.

On the rendered version, the user may use the left side summary (which is collapsible/expandable) to navigate easily through the available endpoints.